cp3pt0-deployment/setup_tenant.sh

```diff
@@ -21,6 +21,8 @@ TETHERED_NS=""
 EXCLUDED_NS=""
 SIZE_PROFILE="starterset"
 INSTALL_MODE="Automatic"
+PREVIEW_MODE=0
+OC_CMD="oc"
```

Comment:
We had `OC=oc` at line 13. Not sure if we introduce `OC_CMD` intentionally?

There are some other places where `${OC} get xxxx` is used, and preview mode should skip them.
```
#  Checking IBMLicensing...
error: unknown flag: --dry-run
See 'oc get --help' for usage.
[✘] User does not have proper permission to get IBMLicensing or IBMLicensing is not installed
```